core-bundle/contao/templates/_new/content_element/table.html.twig

```diff
@@ -10,5 +10,9 @@
 {% endblock -%}
 
 {%- block table_cell_content %}
-    {%- if as_editor_view %}{{ cell|trim|replace({'\n': '<br>\n'})|raw }}{% else %}{{ cell|trim|replace({'\n': '<br>\n'})|insert_tag|raw }}{% endif -%}
+    {%- if as_editor_view %}
+        {{ cell|trim|replace({'\n': '<br>\n'})|raw }}
+    {% else %}
+        {{ cell|trim|replace({'\n': '<br>\n'})|insert_tag|raw }}
+    {% endif -%}
```

Comment:
```suggestion
    {%- if as_editor_view -%}
        {{ cell|trim|replace({'\n': '<br>\n'})|raw }}
    {%- else -%}
        {{ cell|trim|replace({'\n': '<br>\n'})|insert_tag|raw }}
    {%- endif -%}
```

Reply:
Changed in 8bd552f5952b80a01c9ef721233a31191f98e8f0.